react-app/src/components/AppSideBar/AppSideBar.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable complexity */
```

Comment:
If we can't extract things to standalone components then probably we will just ignore it first and open an issue for improvement. Also please use `/* eslint-disable-next-line complexity */` above the function instead of disabling the entire file.